### PR TITLE
Simplify app.yaml files, reduce # of e2e tests, and cleanup for webpa…

### PR DESCRIPTION
**Pull request by jmdobry**
…ck sample.

### CODE REVIEW

**Comment by jmdobry** [comment]
In this PR I scaled back from 9 e2e tests (3 per job) to 3 e2e tests (1 per job). That makes one e2e test each for Node.js stable, 0.12, and 0.10. As you can see the first test passed! https://travis-ci.org/GoogleCloudPlatform/nodejs-docs-samples/jobs/92324990

I also maintained variety in the remaining e2e tests. One deploys Express _and_ promotes it, one deploys Grunt which needs the postinstall hook, and one deploys Koa, which needs the --harmony flag.

File: appengine/express-memcached-session/app.yaml [hunk 1]
```diff
@@ -14,5 +14,4 @@
 runtime: nodejs
 vm: true
 env_variables:
-  PORT: 8080
```
Comment (JustinBeckwith) [inline appengine/express-memcached-session/app.yaml]: Nice!  Is this finally defaulted in the base runtime?

**Comment by JustinBeckwith** [comment]
LGTM

**Comment by jonparrott** [comment]
LGTM.

**Pull request merged**